package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "test": "standard && hallmark && (nyc -s node test.js | faucet) && nyc report",
     "test-browsers-local": "airtap --coverage test.js",
-    "coverage": "nyc report --reporter=text-lcov | coveralls",
```

Comment:
Could you run `npm rm coveralls --save-dev`? LGTM otherwise

Reply:
Removed in 8cc1307